buildpack.md

```diff
@@ -663,31 +663,53 @@ It MUST NOT be used for environment variables that are not defined in this speci
 
 During the build phase, buildpacks MAY write environment variable files to `<layers>/<layer>/env/`,  `<layers>/<layer>/env.build/`, and `<layers>/<layer>/env.launch/` directories.
 
-For each `<layers>/<layer>/` designated as a build layer, for each file written to `<layers>/<layer>/env/` or `<layers>/<layer>/env.build/` by `/bin/build`, the lifecycle MUST modify an environment variable in subsequent executions of `/bin/build`.
+For each `<layers>/<layer>/` designated as a build layer, for each file written to `<layers>/<layer>/env/` or `<layers>/<layer>/env.build/` by `/bin/build`, the lifecycle MUST modify an environment variable in subsequent executions of `/bin/build` according to the modification rules below.
 
-For each file written to `<layers>/<layer>/env.launch/` by `/bin/build`, the lifecycle MUST modify an environment variable when the OCI image is launched.
+For each file written to `<layers>/<layer>/env.launch/` by `/bin/build`, the lifecycle MUST modify an environment variable when the OCI image is launched according to the modification rules below.
 
-The lifecycle MUST set the name of the environment variable to the name of the file up to the first period (`.`) or to the end of the name if no periods are present.
+#### Environment Variable Modification Rules
 
-If the environment variable file name has no period-delimited suffix, then the value of the environment variable MUST be a concatenation of the file contents and the contents of other identically named files delimited by the OS path list separator.
-Within that environment variable value,
+The lifecycle MUST consider the name of the environment variable to be the name of the file up to the first period (`.`) or to the end of the name if no periods are present.
+In all cases, file contents MUST NOT be evaluated by a shell or otherwise modified before inclusion in environment variable values.
+
+##### Delimiter
+
+If the environment variable file name ends in `.delim`, then the value of the environment variable MUST be used to delimit any concatenation within the same layer involving that environment variable.
```

Comment:
The phrase "the value of the environment variable" feels a little weird here to me
How about: 
"If an environment variable file name ends in `.delim`, the file contents MUST be used to delimit..."

Reply:
Thanks, will fix

Reply:
Done :)